automation/src/test/java/org/greenplum/pxf/automation/features/parquet/ParquetWriteTest.java

```diff
@@ -81,11 +83,12 @@ public class ParquetWriteTest extends BaseFeature {
             "bytea_arr            BYTEA[]"      ,         // DataType.BYTEAARRAY
             "char_arr             CHAR(15)[]"   ,         // DataType.BPCHARARRAY
             "varchar_arr          VARCHAR(15)[]",         // DataType.VARCHARARRAY
-            "date_arr             DATE[]"       ,         // DataType.DATEARRAY
-            "numeric_arr          NUMERIC[]"              // DataType.NUMERICARRAY
+            "numeric_arr          NUMERIC[]"    ,         // DataType.NUMERICARRAY
+            "date_arr             DATE[]"                 // DataType.DATEARRAY
     };
 
-    private static final String[] PARQUET_PRIMITIVE_ARRAYS_TABLE_COLUMNS_HIVE = {
+    // CDH (Hive 1.1) does not support date, so we will add the date_arr column as needed in the test case
+    private static ArrayList<String> PARQUET_PRIMITIVE_ARRAYS_TABLE_COLUMNS_HIVE = new ArrayList<>(Arrays.asList(
```

Comment:
```suggestion
    private static List<String> PARQUET_PRIMITIVE_ARRAYS_TABLE_COLUMNS_HIVE = new ArrayList<>(Arrays.asList(
```
try avoiding specific types for variable declarations and use interfaces instead